lib/pymedphys/_experimental/fileformats/mapcheck/core.py

```diff
@@ -37,7 +37,7 @@ def read_mapcheck_txt(file_name):
 
     Mapcheck = namedtuple("Mapcheck", ["x", "y", "dose"])
 
-    with open(file_name, "r") as mapcheck_file:
+    with open(file_name) as mapcheck_file:
```

Comment:
why this change?

Reply:
Added the following pre-commit hook:

https://github.com/asottile/pyupgrade

Reply:
Cool!